gulpfile.js

```diff
@@ -29,7 +30,22 @@ elixir(function(mix) {
     		'public/css/app.css'
     	], {
     		liveCSS: true
-    	});
+    	})
+        .karma({ jsDir:[
```

Comment:
it would be much nicer if we can keep this empty so that it uses the `karma.conf.js` file.. or is it mandatory to pass this here?